easybuild/easyconfigs/p/Pillow/Pillow-3.2.0-foss-2016a-Python-2.7.11.eb

```diff
@@ -0,0 +1,27 @@
+easyblock = 'PythonPackage'
+
+name = 'Pillow'
+version = '3.2.0'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'http://pillow.readthedocs.org/'
+description = """Pillow is the 'friendly PIL fork' by Alex Clark and Contributors.
+ PIL is the Python Imaging Library by Fredrik Lundh and Contributors."""
+
+toolchain = {'name': 'foss', 'version': '2016a'}
+
+source_urls = [PYPI_SOURCE]
+sources = [SOURCE_TAR_GZ]
+
+dependencies = [
+    ('Python', '2.7.11'),
+    ('libjpeg-turbo', '1.4.2','-NASM-2.12.01'),
+    ('libpng', '1.6.21'),
+    ('zlib', '1.2.8'),
+    ('LibTIFF', '4.0.6'),
+    ('freetype', '2.6.3'),
```

Comment:
@boegel is there a real conflict? I don't see an issue locally. The problem is that there are two versions of freetype in the 2016a toolchains, and both versions are in use by many packages. 
The packages that I want to combine have now standardized on freetype 2.6.3.

BTW I perceive this as a symptom of a bigger issue. It would be better to stick to specific library versions for a given toolchain release as much as possible, unless there is an absolute requirement for a different version. 
In this case it would have been better to stick with freetype 2.6.2 all over for the 2016a toolchains.